kubernetes/ama-logs.yaml

```diff
@@ -306,7 +306,7 @@ data:
 # apiVersion: v1
 # kind: Secret
 # metadata:
-#   name: omsagent-aad-msi-token
+#   name: ama-logs-aad-msi-token
```

Comment:
use update to **aad-msi-auth-token** to reflect upcoming AKS side change which aks team working on to get this rolled out.